Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -0,0 +1,1502 @@
+category: Endpoint
+commonfields:
+  id: Carbon Black Enterprise EDR
+  version: -1
+configuration:
+- display: Server URL (e.g. https://defense.conferdeploy.net)
+  name: url
+  required: true
+  type: 0
+- display: Organization Key
+  hidden: false
+  name: organization_key
+  required: true
+  type: 4
+- display: Custom Key
+  name: custom_key
+  required: true
+  type: 4
+- display: Custom ID
+  hidden: false
+  name: custom_id
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: VMware Carbon Black Enterprise EDR is an advanced threat hunting and
+  incident response solution delivering continuous visibility for top security operations
+  centers (SOCs) and incident response (IR) teams.
+display: Carbon Black Enterprise EDR
+name: Carbon Black Enterprise EDR
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The ID of the alert to update.
+      isArray: false
+      name: alert_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Workflow state to filter on. Allowed values
```

Comment:
i think the description here is wrong
dont we update the workflow to this state? if so, why does that say "filter on" ?
no need for allowed values in the description if you provide predefined values

Reply:
Yeah forgot to remove the allowed values, removed now.
Regarding the "filter on". the description is taken from the API docs.